app/models/conference.rb

```diff
@@ -15,4 +17,15 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      if [starts_on, ends_on].count(&:nil?) == 1
```

Comment:
I believe that the reasons for presence validations on the dates itself, is only to make our dev life easier, and not per se helping either the user or data integrity. And, on the other hand, I can imagine that orga's life can become harder if it is never ever allowed to add a conference without dates. So, with those considerations, I opted for this solution. But I am not per se against the presence validations, if you think that other considerations are more important?